pyaedt/modeler/cad/Primitives.py

```diff
@@ -275,45 +587,4908 @@ def version(self):
+                        name = data["Attributes"]["Name"]
+                        props["ID"] = data["ID"]
+                        props["Type"] = data["GeometryEntityListParameters"]["EntityType"]
+                        if props["Type"] == "Object":
+                            name_list = []
+                            for element in data["GeometryEntityListParameters"]["EntityList"]:
+                                element_name = self.oeditor.GetObjectNameByID(int(element))
+                                name_list.append(element_name)
+                            props["List"] = name_list
+                        else:
+                            props["List"] = data["GeometryEntityListParameters"]["EntityList"]
+                        design_lists.append(Lists(self, props, name))
+            except:
+                self.logger.info("Lists were not retrieved from AEDT file")
+        return design_lists
+
+    def _get_planes_data(self):
+        """Retrieve planes data.
+
+        Returns
+        -------
+        [Dict with List information]
+        """
+        try:
+            return {
+                plane_name: self.oeditor.GetChildObject(plane_name)
+                for plane_name in self.oeditor.GetChildNames("Planes")
+            }
+        except TypeError:
+            return {}
+
+    def __get__(self, instance, owner):
+        self._app = instance
+        return self
+
+    @pyaedt_function_handler()
+    def fit_all(self):
+        """Fit all.
+
+        References
+        ----------
+
+        >>> oEditor.FitAll
+        """
+        self.oeditor.FitAll()
+
+    @pyaedt_function_handler()
+    def _find_perpendicular_points(self, face):
+        if isinstance(face, str):
+            vertices = [i.position for i in self[face].vertices]
+        else:
+            vertices = []
+            for vertex in list(self.oeditor.GetVertexIDsFromFace(face)):
+                vertices.append([float(i) for i in list(self.oeditor.GetVertexPosition(vertex))])
+        assert len(vertices) > 2, "Automatic A-B Assignment can be done only on face with more than 2 vertices."
+        origin = vertices[0]
+        a_end = []
+        b_end = []
+        tol = 1e-10
+        for v in vertices[1:]:
+            edge1 = GeometryOperators.v_points(origin, v)
+            for v2 in vertices[1:]:
+                if v2 != v:
+                    edge2 = GeometryOperators.v_points(origin, v2)
+                    if abs(GeometryOperators.v_dot(edge1, edge2)) < tol:
+                        a_end = v
+                        b_end = v2
+                        break
+            if a_end:
+                break
+        if not a_end:
+            a_end = vertices[1]
+            b_end = vertices[2]
+            return False, (origin, a_end, b_end)
+        return True, (origin, a_end, b_end)
+
+    @pyaedt_function_handler()
+    def cover_lines(self, selection):
+        """Cover a closed line and transform it to sheet.
```

Comment:
The function has plural lines. Should it be "Cover closed lines and transform them to a sheet"?

Reply:
```suggestion
        """Cover closed lines and transform them to a sheet
```